index.html

```diff
@@ -1,243 +1,361 @@
+          <div class="logo">Timothy</div>
+        </a>
+        <button type="button" class="hamburger-menu">
+          <img src="./assets/images/vectors/Icon - Menu.png" alt="" />
+        </button>
+        <nav class="flex-nav">
+          <div class="background-filter"></div>
+          <div class="nav-items-wrapper">
+            <ul>
+              <a href="#portfolio"><li>Portfolio</li></a>
+              <a href="#about"><li>About</li></a>
+              <a href="#contact"><li>Contact</li></a>
+            </ul>
+            <span class="close">+</span>
+          </div>
+        </nav>
+      </header>
 
-<body>
-  <div class="wrapper">
-    <header class="header">
-      <a href="#home">
-        <div class="logo">Timothy</div>
-      </a>
-      <nav class="hamburger-menu">
-        <img src="./assets/images/vectors/Icon - Menu.png" alt="">
-      </nav>
-      <nav class="flex-nav">
-        <div class="background-filter"></div>
-        <div class="nav-items-wrapper">
-          <ul>
-            <a href="#portfolio"><li>Portfolio</li></a>
-            <a href="#about"><li>About</li></a>
-            <a href="#contact"><li>Contact</li></a>
-          </ul>
-          <span class="close">+</span>
+      <main class="main-content">
+        <div class="hero-wrapper">
+          <section class="hero" id="home">
+            <h1 class="section-header">I’m Timothy Glad to see you!</h1>
+            <p class="intro">
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <span class="lets-connect">LET’S CONNECT</span>
+            <ul class="socials">
+              <li>
+                <a
+                  href="https://twitter.com/njorogemwangi21"
+                  target="_blank"
+                  rel="noopener"
+                  ><img src="./assets/images/vectors/twitter.svg" alt=""
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="https://linkedin.com/in/timothy-njoroge-mwangi"
+                  target="_blank"
+                  rel="noopener"
+                  ><img src="./assets/images/vectors/Linkedin.svg" alt=""
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="https://github.com/simplegoose"
+                  target="_blank"
+                  rel="noopener"
+                  ><img src="./assets/images/vectors/github.svg" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"></a
+                ><img src="./assets/images/vectors/M.svg" alt="" />
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./assets/images/vectors/chill.svg" alt=""
```

Comment:
- I can see that you have images with some missing `alt text` here. Please ensure that all images have an `alt` text. Remember to include the links to the changes you have made in the PR report 👍 👍